pkg/api/handlers/compat/containers_create.go

```diff
@@ -308,7 +308,7 @@ func cliOpts(cc handlers.CreateContainerConfig, rtc *config.Config) (*entities.C
 	}
 
 	// sigh docker-compose sets the mac address on the container config instead on the per network endpoint config
-	containerMacAddress := cc.MacAddress
+	containerMacAddress := cc.MacAddress //nolint:staticcheck
```

Comment:
why nolint?

Reply:
It wants to use another type that isn't part of the call stack at all.  I refrained from turning this into a bigger journey.

Reply:
I think this should still be used for backwards compat.  I'll drop a comment.